Three/include/CGAL/Three/Three.h

```diff
@@ -29,8 +29,15 @@
 #  define THREE_EXPORT Q_DECL_IMPORT
 #endif
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#define SkipEmptyParts QString::SkipEmptyParts
+#else
+#define SkipEmptyParts ::Qt::SkipEmptyParts
+#endif
```

Comment:
Does not forget CGAL dev manual:
https://doc.cgal.org/latest/Manual/devman_code_format.html
> Names of macros should begin with the prefix `CGAL_`.